src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarExtraLinks.tsx

```diff
@@ -150,7 +154,9 @@ ArtworkSidebarExtraLinksContainerProps
       return (
         <Container>
           Have a question?{" "}
-          <Link onClick={this.onClickBuyNowFAQ.bind(this)}>Read our FAQ</Link>{" "}
+          <Link onClick={this.onClickBuyNowFAQ.bind(this)}>
+            Visit our help center
```

Comment:
Noticed that we [track clicks for Buy Now FAQ link](https://github.com/artsy/force/blob/1ac011e6d9b2a9b6503bc9ecada3b39260c8cf09/src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarExtraLinks.tsx#L52-L59) and Collector FAQ link below. I assume the CTA copy changes _may_ effect how users interact with the page. Do we want to modify the tracking or are we fine keeping the existing tracking as is? cc @anipetrov

Reply:
let's change the `label` property to the new CTA please!